axon-framework/serialization.md

```diff
@@ -214,3 +212,25 @@ public class SerializerConfiguration {
     }
 }
 ```
+
+### ContentTypeConverters
+
+An [upcaster](events/event-versioning.md#event-upcasting) works on a given content type \(e.g. dom4j Document\). 
+To provide extra flexibility between upcasters, content types between chained upcasters may vary. 
+Axon will try to convert between the content types automatically by using `ContentTypeConverter`s. 
```

Comment:
```suggestion
Axon will try to convert between the content types automatically by using `ContentTypeConverter`. 
```
I know you meant it to be plural but I don't think we'd need the "'s"  for it. I'll change the rest as well, but please change back if you feel strongly about it. :)